cql3/statements/cf_prop_defs.cc

```diff
@@ -78,7 +78,7 @@ const sstring cf_prop_defs::COMPACTION_STRATEGY_CLASS_KEY = "class";
 
 const sstring cf_prop_defs::COMPACTION_ENABLED_KEY = "enabled";
 
-void cf_prop_defs::validate(const db::extensions& exts) const {
+void cf_prop_defs::validate(const database& db) const {
```

Comment:
I'm sorry, I didn't understand this patch. Why is it important that the "extensions is taken from the global database"? In a previous version of this patch, you had fishy caching here, so it was dangerous if called twice with different extensions, but I think you don't have this caching any more (at least not in the form you had it). So why do we need this patch?

Reply:
Ok, I see, sorry, I didn't update the commit description. This patch was useful in resolving one of the review comments ([this comment](https://github.com/scylladb/scylla/pull/5908#discussion_r387513500), fixed by the [next commit](https://github.com/scylladb/scylla/pull/5908/commits/0d4cc56d981af19e05732fb5ffb03f77e72d89b7)).

I updated the description.